app/menu.js

```diff
@@ -105,7 +105,7 @@ module.exports = ({createWindow, updatePlugins}) => {
         type: 'separator'
       },
       {
-        label: 'Close',
+        label: 'Close Tab',
```

Comment:
This also closes split panes now. Maybe something like `Close Session` or `Close Pane` could work? Not sure.